<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>聊天室</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 先把默认的边框样式设好（如果需要） */
QTextEdit, QTextBrowser {
    border: 1px solid #ccc;
    border-radius: 4px;
}

/* 获得焦点时，不要再画那条蓝色框 */
QTextEdit:focus, QTextBrowser:focus {
    border: 1px solid #aaa;    /* 或者直接 border: none; */
    outline: none;              /* 有些样式引擎会识别 outline */
}
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QTextEdit" name="clientTextEdit">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>430</y>
      <width>770</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,1);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>565</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid blue;         /* 正常状态边框 */
    background-color: white;        /* 正常状态背景色 */
    color: black;                   /* 文字颜色 */
    padding: 6px 12px;              /* 内边距 */
}
QPushButton:hover {
    background-color: #303F9F;       /* 悬停时加深背景 */
}
QPushButton:pressed {
    background-color: #e0e0e0;      /* 按下时背景色（浅灰色示例） */
    border: 2px solid #0000ff;      /* 按下时边框色（深蓝示例） */
}
</string>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serveipLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>15</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* 设置元素边框：宽度 2px，凹槽样式（groove），灰色 */
border: 2px groove gray;

/* 设置元素的圆角半径：所有角均为 10px */
border-radius: 10px;

/* 设置元素内边距：上下各 2px，左右各 4px */
padding: 2px 4px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>输入服务器IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="portlabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>15</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>端口：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>15</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* 设置元素边框：宽度 2px，凹槽样式（groove），灰色 */
border: 2px groove gray;

/* 设置元素的圆角半径：所有角均为 10px */
border-radius: 10px;

/* 设置元素内边距：上下各 2px，左右各 4px */
padding: 2px 4px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>输入端口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serveButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>15</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3F51B5;       /* 按钮背景色 */
    color: white;                    /* 文字颜色 */
	/* 设置元素边框：宽度 2px，凹槽样式（groove），灰色 */
	border: 2px groove gray;
	/* 设置元素的圆角半径：所有角均为 10px */
	border-radius: 10px;
	/* 设置元素内边距：上下各 2px，左右各 4px */
	padding: 2px 4px;
}
QPushButton:hover {
    background-color: #303F9F;       /* 悬停时加深背景 */
}
QPushButton:pressed {
    background-color: #e0e0e0;      /* 按下时背景色（浅灰色示例） */
    border: 2px solid #0000ff;      /* 按下时边框色（深蓝示例） */
}
</string>
    </property>
    <property name="text">
     <string>连接服务器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectfileButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3F51B5;       /* 按钮背景色 */
    color: white;                    /* 文字颜色 */
	/* 设置元素边框：宽度 2px，凹槽样式（groove），灰色 */
	border: 2px groove gray;
	/* 设置元素的圆角半径：所有角均为 10px */
	border-radius: 10px;
	/* 设置元素内边距：上下各 2px，左右各 4px */
	padding: 2px 4px;
}
QPushButton:hover {
    background-color: #303F9F;       /* 悬停时加深背景 */
}
QPushButton:pressed {
    background-color: #e0e0e0;      /* 按下时背景色（浅灰色示例） */
    border: 2px solid #0000ff;      /* 按下时边框色（深蓝示例） */
}
</string>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="servefileButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #3F51B5;       /* 按钮背景色 */
    color: white;                    /* 文字颜色 */
	/* 设置元素边框：宽度 2px，凹槽样式（groove），灰色 */
	border: 2px groove gray;
	/* 设置元素的圆角半径：所有角均为 10px */
	border-radius: 10px;
	/* 设置元素内边距：上下各 2px，左右各 4px */
	padding: 2px 4px;
}
QPushButton:hover {
    background-color: #303F9F;       /* 悬停时加深背景 */
}
QPushButton:pressed {
    background-color: #e0e0e0;      /* 按下时背景色（浅灰色示例） */
    border: 2px solid #0000ff;      /* 按下时边框色（深蓝示例） */
}
</string>
    </property>
    <property name="text">
     <string>发送文件</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="fileBrowser">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="serveBrowser">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>60</y>
      <width>480</width>
      <height>340</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="filewidget" native="true">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>60</y>
      <width>290</width>
      <height>340</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <widget class="QScrollArea" name="filescrollArea">
     <property name="geometry">
      <rect>
       <x>495</x>
       <y>60</y>
       <width>290</width>
       <height>340</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: lightgray;</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>288</width>
        <height>338</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="fileicolabel1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="filenamelabel1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>145</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>文件名:</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="filenamelabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>145</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>文件名:</string>
     </property>
    </widget>
    <widget class="QLabel" name="fileicolabel2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="filenamelabel3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>145</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>文件名:</string>
     </property>
    </widget>
    <widget class="QLabel" name="fileicolabel3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextBrowser" name="servefileBrowse">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>290</width>
       <height>340</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="acceptButton1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;               /* 去除边框 */
    background-color: transparent; /* 透明背景 */
    color: #0066cc;              /* 文本颜色 */
}
QPushButton:hover {
    color: #0088ff;              /* 鼠标悬停时文本颜色 */
}
</string>
     </property>
     <property name="text">
      <string>接受</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refuseButton1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;               /* 去除边框 */
    background-color: transparent; /* 透明背景 */
    color: #0066cc;              /* 文本颜色 */
}
QPushButton:hover {
    color: #0088ff;              /* 鼠标悬停时文本颜色 */
}
</string>
     </property>
     <property name="text">
      <string>拒绝</string>
     </property>
    </widget>
    <widget class="QPushButton" name="acceptButton2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;               /* 去除边框 */
    background-color: transparent; /* 透明背景 */
    color: #0066cc;              /* 文本颜色 */
}
QPushButton:hover {
    color: #0088ff;              /* 鼠标悬停时文本颜色 */
}
</string>
     </property>
     <property name="text">
      <string>接受</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refuseButton2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;               /* 去除边框 */
    background-color: transparent; /* 透明背景 */
    color: #0066cc;              /* 文本颜色 */
}
QPushButton:hover {
    color: #0088ff;              /* 鼠标悬停时文本颜色 */
}
</string>
     </property>
     <property name="text">
      <string>拒绝</string>
     </property>
    </widget>
    <widget class="QPushButton" name="acceptButton3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>300</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;               /* 去除边框 */
    background-color: transparent; /* 透明背景 */
    color: #0066cc;              /* 文本颜色 */
}
QPushButton:hover {
    color: #0088ff;              /* 鼠标悬停时文本颜色 */
}
</string>
     </property>
     <property name="text">
      <string>接受</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refuseButton3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>300</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;               /* 去除边框 */
    background-color: transparent; /* 透明背景 */
    color: #0066cc;              /* 文本颜色 */
}
QPushButton:hover {
    color: #0088ff;              /* 鼠标悬停时文本颜色 */
}
</string>
     </property>
     <property name="text">
      <string>拒绝</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveasButton2">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>190</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;               /* 去除边框 */
    background-color: transparent; /* 透明背景 */
    color: #0066cc;              /* 文本颜色 */
}
QPushButton:hover {
    color: #0088ff;              /* 鼠标悬停时文本颜色 */
}
</string>
     </property>
     <property name="text">
      <string>另存为</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveasButton1">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>80</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;               /* 去除边框 */
    background-color: transparent; /* 透明背景 */
    color: #0066cc;              /* 文本颜色 */
}
QPushButton:hover {
    color: #0088ff;              /* 鼠标悬停时文本颜色 */
}
</string>
     </property>
     <property name="text">
      <string>另存为</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveasButton3">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>300</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;               /* 去除边框 */
    background-color: transparent; /* 透明背景 */
    color: #0066cc;              /* 文本颜色 */
}
QPushButton:hover {
    color: #0088ff;              /* 鼠标悬停时文本颜色 */
}
</string>
     </property>
     <property name="text">
      <string>另存为</string>
     </property>
    </widget>
    <widget class="QLabel" name="filesizelabel1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="filesizelabel2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="filesizelabel3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>230</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>servefileBrowse</zorder>
    <zorder>fileicolabel1</zorder>
    <zorder>filenamelabel1</zorder>
    <zorder>progressBar1</zorder>
    <zorder>progressBar2</zorder>
    <zorder>filenamelabel2</zorder>
    <zorder>fileicolabel2</zorder>
    <zorder>progressBar3</zorder>
    <zorder>filenamelabel3</zorder>
    <zorder>fileicolabel3</zorder>
    <zorder>filescrollArea</zorder>
    <zorder>acceptButton1</zorder>
    <zorder>refuseButton1</zorder>
    <zorder>acceptButton2</zorder>
    <zorder>refuseButton2</zorder>
    <zorder>acceptButton3</zorder>
    <zorder>refuseButton3</zorder>
    <zorder>saveasButton2</zorder>
    <zorder>saveasButton1</zorder>
    <zorder>saveasButton3</zorder>
    <zorder>filesizelabel1</zorder>
    <zorder>filesizelabel2</zorder>
    <zorder>filesizelabel3</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
